channels/auth.py

```diff
@@ -20,7 +22,7 @@
 
 
 @database_sync_to_async
-def get_user(scope):
+def get_user(scope: Dict[str, Any]) -> Awaitable[Union[User, AnonymousUser]]:
```

Comment:
That isn't necessarily true, e.g. when using a custom `User` model.

Reply:
Okay, thanks, will use that type for every user typing from now on

Reply:
Can I use AbstractUser instead? Since ```CustomUser``` and ```User``` inherit from it?

Reply:
I'd use `django.contrib.auth.base_user.AbstractBaseUser` for this.

Reply:
Yes, you're right

Reply:
@carltongibson here